utils/process.py

```
@@ -0,0 +1,49 @@
+"""Process management utilities."""
+
+import subprocess
+import time
+from typing import Optional
```

Ruff (0.8.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)